### PR TITLE
Common packages

### PR DESCRIPTION
**Pull request by trustmaster**
This adds basic packages that seem common among our Go projects and are known to be (more or less) good and proved over time.

I'm not sure about [Gorilla/Schema](http://www.gorillatoolkit.org/pkg/schema), I used it quite some time ago but I suspect that there may be better solutions nowadays.

### CODE REVIEW

**Comment by bastiankoetsier** [comment]
+1
I would leave out `schema` because afaik we did not really test this here at HF yet (mostly API-based projects)

**Comment by trustmaster** [comment]
@bastiankoetsier I've added it because I've noticed people parsing URL query strings into structs manually for GET requests (and vice versa). So I think it's good for people to know there are better solutions, I'm just not sure this particular package is the best, it's just what I've worked with.

P.S.: there's no Concourse in this repo, so GitHub Review button is needed for approval 😉

File: packages.md [hunk 1]
```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
...
+- [github.com/mattes/migrate](https://github.com/mattes/migrate) - database migrations
+
+## Forms
+
+- [github.com/gorilla/schema](https://github.com/gorilla/schema) - decode forms and urls into Go structs and vice versa
+
+## HTTP
+
+- [github.com/go-chi/chi](https://github.com/go-chi/chi) - router with decent performance and middleware support
+- [github.com/dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) - JSON Web Token library
```
Comment (vgarvardt) [inline packages.md:29]: Please use https://github.com/hellofresh/jwt-auth-go here - it wraps `github.com/dgrijalva/jwt-go` with HF-specific stuff

File: packages.md [hunk 2]
```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
 
 If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
 
+## Configuration
+
+- [github.com/kelseyhightower/envconfig](https://github.com/kelseyhightower/envconfig) - get configs from environment variables
+
```
Comment (vgarvardt) [inline packages.md]: Please add `github.com/spf13/viper` here - normally we use this in our projects and `github.com/kelseyhightower/envconfig` is used by viper for environment backend.

File: packages.md [hunk 3]
```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
...
+- [github.com/rafaeljesus/retry-go](https://github.com/rafaeljesus/retry-go) - retry mechanism to build into your HTTP clients
+- [github.com/sony/gobreaker](https://github.com/sony/gobreaker) - CircuitBreaker implementation, use this in your HTTP clients
+
+
+## Logging and Metrics
+
+- [github.com/sirupsen/logrus](https://github.com/sirupsen/logrus) - logging package
+- [github.com/hellofresh/logging-go](https://github.com/hellofresh/logging-go) - HelloFresh extensions to Logrus
+- [github.com/hellofresh/stats-go](https://github.com/hellofresh/stats-go) - collect and push metrics
+
```
Comment (vgarvardt) [inline packages.md]: Please add `github.com/hellofresh/gcloud-opentracing` and `github.com/opentracing/opentracing-go` - tools that we use for distributed tracing.

**Comment by trustmaster** [comment]
@vgarvardt thanks! Added them all, except for `github.com/hellofresh/jwt-auth-go` which is a private repo. So silly we can't link to them consistently.

**Review by vgarvardt — APPROVED** [review]

**Comment by rafaeljesus** [comment]
👍

**Pull request merged**